src/components/projekti/nahtavillaolo/kuulutuksentiedot/KuulutuksenTiedot.tsx

```diff
@@ -75,9 +75,9 @@ export default function KuulutuksenTiedot({ kirjaamoOsoitteet }: Props) {
           kirjaamoOsoitteet={kirjaamoOsoitteet}
           nahtavillaoloVaihe={projekti?.nahtavillaoloVaihe}
         />
-        <KuulutuksenJaIlmoituksenEsikatselu />
-        <NahtavillaoloPainikkeet />
       </form>
+      <KuulutuksenJaIlmoituksenEsikatselu projekti={projekti as Projekti} />
+      <NahtavillaoloPainikkeet />
     </FormProvider>
   );
 }
```

Comment:
Konsoliin tuli herjaa, jos nämä olivat formin sisällä. KuulutuksenJaIlmoituksenEsikatselu-komponentissa on tuo jännä systeemi, millä saa pdf:t näkyviin, ja se on kai pohjimmiltaan jonkinlainen form-submit-systeemi, ja tuli herjaa, että ei saa olla formia formin sisällä.

Reply:
Joo. Siis se kai clashaa jotenki pahasti ku on kaks FormProvideria sisäkkäin

Reply:
Joo. Siis se kai clashaa jotenki pahasti ku on kaks FormProvideria sisäkkäin